packages/core/.env.local

```diff
@@ -0,0 +1,3 @@
+USE_LOCAL_DB = true
+DB_NAME = jkv2
```

Comment:
this shouldn't be in git - should be ignored

Reply:
Then why not put those default settings in .env or .env.development?
Please read this doc https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env